R/auto.R

```diff
@@ -291,6 +293,17 @@ check_post_release <- function() {
 
 gh_scopes <- function() {
   out <- attr(gh::gh("/user"), "response")$"x-oauth-scopes"
-  if (out == "") return(character())
+  if (out == "") {
+    return(character())
+  }
   strsplit(out, ", *")[[1]]
 }
+
+check_gitignore <- function(files) {
+  files_ignored <- map_chr(files, ~ system2("git", c("check-ignore", .x), stdout = TRUE))
+  if (length(files_ignored >= 1)) {
```

Comment:
What are we checking here?

Reply:
`files_ignored` stores names of files which are listed in `.gitignore` but shouldn`t be there.